### ISSUE TITLE
Table formatting appears to be broken when table cells are empty

### ISSUE THREAD

**Issue by morgen-peschke** [piotrmurach/tty-markdown#22]
### Describe the problem
Formatting documents with tables containing an empty cell fails.

### Steps to reproduce the problem
For completeness, I've saved a script with the variations I've tried, which can be found [here](https://github.com/morgen-peschke/dockerfiles/blob/master/tty-markdown/reproduce-error.sh).

The minimal input which triggers this is:
```
| a |
|---|
|   |
```

The minimal input which successfully passes is:
```
| a      |
|--------|
| &nbsp; |
``` 

### Actual Behavior
Minimal failing case produces this error:
```
/usr/local/bundle/gems/tty-markdown-0.6.0/lib/tty/markdown/parser.rb:370:in `max': comparison of NilClass with 1 failed (ArgumentError)
	from /usr/local/bundle/gems/tty-markdown-0.6.0/lib/tty/markdown/parser.rb:370:in `max_width'
	from /usr/local/bundle/gems/tty-markdown-0.6.0/lib/tty/markdown/parser.rb:386:in `block in max_widths'
	from /usr/local/bundle/gems/tty-markdown-0.6.0/lib/tty/markdown/parser.rb:385:in `each'
	from /usr/local/bundle/gems/tty-markdown-0.6.0/lib/tty/markdown/parser.rb:385:in `each_with_index'
	from /usr/local/bundle/gems/tty-markdown-0.6.0/lib/tty/markdown/parser.rb:385:in `each'
	from /usr/local/bundle/gems/tty-markdown-0.6.0/lib/tty/markdown/parser.rb:385:in `reduce'
	from /usr/local/bundle/gems/tty-markdown-0.6.0/lib/tty/markdown/parser.rb:385:in `max_widths'
	from /usr/local/bundle/gems/tty-markdown-0.6.0/lib/tty/markdown/parser.rb:251:in `border'
	from /usr/local/bundle/gems/tty-markdown-0.6.0/lib/tty/markdown/parser.rb:232:in `convert_thead'
	from /usr/local/bundle/gems/tty-markdown-0.6.0/lib/tty/markdown/parser.rb:30:in `convert'
	from /usr/local/bundle/gems/tty-markdown-0.6.0/lib/tty/markdown/parser.rb:43:in `block in inner'
	from /usr/local/bundle/gems/tty-markdown-0.6.0/lib/tty/markdown/parser.rb:38:in `each'
	from /usr/local/bundle/gems/tty-markdown-0.6.0/lib/tty/markdown/parser.rb:38:in `each_with_index'
	from /usr/local/bundle/gems/tty-markdown-0.6.0/lib/tty/markdown/parser.rb:38:in `inner'
	from /usr/local/bundle/gems/tty-markdown-0.6.0/lib/tty/markdown/parser.rb:224:in `convert_table'
	from /usr/local/bundle/gems/tty-markdown-0.6.0/lib/tty/markdown/parser.rb:30:in `convert'
	from /usr/local/bundle/gems/tty-markdown-0.6.0/lib/tty/markdown/parser.rb:43:in `block in inner'
	from /usr/local/bundle/gems/tty-markdown-0.6.0/lib/tty/markdown/parser.rb:38:in `each'
	from /usr/local/bundle/gems/tty-markdown-0.6.0/lib/tty/markdown/parser.rb:38:in `each_with_index'
	from /usr/local/bundle/gems/tty-markdown-0.6.0/lib/tty/markdown/parser.rb:38:in `inner'
	from /usr/local/bundle/gems/tty-markdown-0.6.0/lib/tty/markdown/parser.rb:49:in `convert_root'
	from /usr/local/bundle/gems/tty-markdown-0.6.0/lib/tty/markdown/parser.rb:30:in `convert'
	from /usr/local/bundle/gems/kramdown-1.16.2/lib/kramdown/converter/base.rb:105:in `convert'
	from /usr/local/bundle/gems/tty-markdown-0.6.0/lib/tty/markdown.rb:92:in `parse'
	from /tty-markdown.rb:2:in `<main>'
```
Minimal passing case produces this output:
```
┌───┐
│ a │
├───┤
│   │
└───┘
```

### Expected Behavior
Both minimal inputs should produce the same table:
```
┌───┐
│ a │
├───┤
│   │
└───┘
```

### Describe your environment
I'm attempting to create a docker image for a minimal wrapper around tty-markdown:
```ruby
require 'tty-markdown'
puts TTY::Markdown.parse STDIN.read
```

The complete definition can be found [here](https://github.com/morgen-peschke/dockerfiles/tree/master/tty-markdown).

The Dockerfile is extremely minimal, and inherits from the standard Ruby image:
```Dockerfile
FROM ruby:2.7
RUN gem install tty-markdown
COPY tty-markdown.rb version.sh /
ENTRYPOINT ["ruby", "-W0", "/tty-markdown.rb"]
```

Currently, it's pulling version 0.6.0 of TTY::Markdown

**Comment by piotrmurach** [comment]
Sorry for the late reply! I do appreciate this fantastic issue report which made the bug easy to squash.

**Comment by morgen-peschke** [comment]
Great to hear! Do you have an ETA for the next release?

**Comment by piotrmurach** [comment]
I've been working constantly on the new release for the past week or so. There are many things that needed doing and there are few remaining. This is my sole focus until release though I don't want to pin down any dates. It will be released soon.

**Comment by morgen-peschke** [comment]
Makes sense, thanks for maintaining this :)

**Comment by piotrmurach** [comment]
Released `v0.7.0` with many updates(see [changelog](https://github.com/piotrmurach/tty-markdown/blob/master/CHANGELOG.md)). I hope you enjoy it!

**Comment by morgen-peschke** [comment]
Works great! Thanks 👍
